packages/graphiql/src/components/GraphiQL.tsx

```diff
@@ -304,56 +310,57 @@ class GraphiQLInternals extends React.Component<
       //   onDoubleClick={this.handleResetResize}
       //   onMouseDown={this.handleResizeStart}>
       //   <div className="queryWrap" style={queryWrapStyle}>
-      <QueryEditor
-        onHintInformationRender={this.handleHintInformationRender}
-        onClickReference={this.handleClickReference}
-        editorTheme={this.props.editorTheme}
-        readOnly={this.props.readOnly}
-        editorOptions={this.props.operationEditorOptions}
-      />
+      <section>
+        <EditorTabs tabs={[ `Query`, `Explorer` ]}>
```

Comment:
@harshithpabbati i see what you're doing here, fantastic. this is a great way to start - statically. In a later iteration, or maybe the next few commits, we can make this a bit more dynamic and user configurable.

Reply:
Maybe a file which consists of all the data in it and mapping it over here works?

Reply:
for starters, we will use props to pass configuration down from `GraphiQL` component props. so, i was thinking a `sessionTabs` property

other notes:

- lets use `SessionTabs` instead of `EditorTabs`, as some of these tabs will not be editor instances once users configure them. the session tabs will have a unique state for each of these tabs
- `Query` -> `Operation`, ideally where possible, especially with new code, except when referring to an operation with an operationType of query. I haven't renamed the `QueryEditor` component but I want to. Just an official GraphQL spec language thing

Reply:
@harshithpabbati updated the above comment - what do you think? or should we just keep it more design focused for now and introduce this in the next PR?

Reply:
I guess we can introduce that in the next PR

Reply:
But, this PR has an issue. Editor in the Query tab is having default height of 5px. I am not able to figure out why it is happening.

Reply:
i wonder if it's lingering css files we need to remove

Reply:
ohhhh the monaco editor itself right